lua/packer.lua

```diff
@@ -268,17 +268,19 @@ packer.install = function(...)
   local install_plugins
   if ... then
     install_plugins = {...}
-  else
-    install_plugins = plugin_utils.find_missing_plugins(plugins)
   end
+  async(function()
+    if not install_plugins then
```

Comment:
Dumb Lua question: is `install_plugins` reliably falsey without explicit initialization to `nil`? I assume so?

Reply:
@wbthomason I've done this quite a lot before, you don't actually need to explicitly assign to nil, any random variable reference will being assigned to nil (the local bit makes it local but would otherwise be a global nil value).